packages/web-react/src/components/Dropdown/useDropdownAriaProps.ts

```diff
@@ -0,0 +1,36 @@
+import { Booleanish, ClickEvent } from '../../types';
+
+const NAME_ARIA_EXPANDED = 'aria-expanded';
+const NAME_ARIA_CONTROLS = 'aria-controls';
+
+export interface UseDropdownAriaPropsProps {
```

Comment:
I'm not really quite sure about this naming. It's unclear how to understand this.
First what came on my mind was the line: "It's a bird! It's a plane! It's Superman!"

Reply:
Ok, i want to name properties for hook, which is named as Props. So, it is `Props` + `UseDropdownAriaProps` = `UseDropdownAriaPropsProps`. 
If you have better name, please, let me know